src/apis/modTools.es6.js

```diff
@@ -24,8 +24,14 @@ const approve = (apiOptions, fullname) => {
 
 const distinguish = (apiOptions, fullname, distinguishType) => {
   // Distinguish a link or comment
+
+  const distinguishTypeMap = {
+    'moderator': 'yes',
+    '': 'no',
+  };
+
   const body = {
-    how: distinguishType,
+    how: distinguishTypeMap[distinguishType],
```

Comment:
Should we add a check here to ensure that the passed `distinguishType` is valid?